Sample/Unity Sample Application/Assets/TitleGUI.cs

```diff
@@ -42,67 +57,56 @@ public class TitleGUI : MonoBehaviour {
 		else
 			Vungle.onResume();
 	}
-	
-	void OnGUI () {
-		GUILayout.BeginArea (new Rect (0, 0, Screen.width, Screen.height));
-		List<string> list = new List<string>(placements.Keys);
-
-		GUI.enabled = true;
-
-		GUILayout.Label ("AppID " + appID, titleLabelStyle);
-		if (GUILayout.Button ("Init SDK")) {
-			string[] array = new string[placements.Keys.Count];
-			placements.Keys.CopyTo(array, 0);
-			Vungle.init (appID, appID, appID, array);
-			initializeEventHandlers ();
-		}
-
-		GUI.enabled = adInited;
-		GUILayout.Label ("Placement 1");
-		GUILayout.Label ("PlacementID " + list[0]);
-		GUI.enabled = placements[list[0]];
-		if (GUILayout.Button ("play")) {
-			Vungle.playAd(list[0]);
-		}
-		GUI.enabled = adInited;
-
-		GUILayout.Label ("Placement 2");
-		GUILayout.Label ("PlacementID " + list[1]);
-		GUI.enabled = placements[list[1]];
-		GUILayout.BeginHorizontal ();
-		if (GUILayout.Button ("play")) {
-			Vungle.playAd(list[1]);
-		}
-		GUI.enabled = !placements[list[1]];
-		if (GUILayout.Button ("load")) {
-			Vungle.loadAd(list[1]);
-		}
-		GUILayout.EndHorizontal();
-		GUI.enabled = adInited;
-
-		GUILayout.Label ("Placement 3");
-		GUILayout.Label ("PlacementID " + list[2]);
-		GUI.enabled = placements[list[2]];
-		GUILayout.BeginHorizontal ();
-		if (GUILayout.Button ("play")) {
-			Vungle.playAd(list[2]);
-		}
-		GUI.enabled = !placements[list[2]];
-		if (GUILayout.Button ("load")) {
-			Vungle.loadAd(list[2]);
-		}
-		GUILayout.EndHorizontal();
-
-		GUILayout.EndArea ();
+
+	void SetupButtonsAndText () {
+		placementIdList = new List<string>(placements.Keys);
+
+		appIDText.text = "App ID: " + appID;
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
TitleGUI.SetupButtonsAndText () (at Assets/TitleGUI.cs:64)
TitleGUI.Start () (at Assets/TitleGUI.cs:45)